compiler/hie-core/src/Development/IDE/Core/Rules.hs

```diff
@@ -119,6 +120,20 @@ getDefinition file pos = fmap join $ runMaybeT $ do
 getParsedModule :: NormalizedFilePath -> Action (Maybe ParsedModule)
 getParsedModule file = use GetParsedModule file
 
+-- | Get module interfaces.
+getModIfaces ::
+       NormalizedFilePath -> Action (Maybe [(FilePath, ModIface, HieFile)])
```

Comment:
This could do with a separate ADT for the return type and a comment on that ADT what the filepath is.